templates/openstack/what-is-openstack.html

```diff
@@ -465,325 +578,288 @@ <h2 class="u-sv3">OpenStack is big</h2>
-            <p>
-              A new version of OpenStack comes every six months; Canonical
-              provides support for all new versions within two weeks from the
-              upstream release.
-            </p>
-          </div>
+        <li class="p-list__item">
+          <h3 class="p-heading--5">Every OpenStack version supported</h3>
+          <p>
+            A new version of OpenStack comes every six months; Canonical provides support for all new versions within two weeks from the upstream release.
+          </p>
         </li>
-        <li class="p-matrix__item">
-          <div class="p-matix__content">
-            <h5>Clear and fully automated upgrade path</h5>
-            <p>
-              Users can upgrade between consecutive OpenStack versions in a
-              fully automated way, benefitting from new features brought by the
-              latest release.
-            </p>
-          </div>
+        <li class="p-list__item">
+          <h3 class="p-heading--5">Clear and fully automated upgrade path</h3>
+          <p>
+            Users can upgrade between consecutive OpenStack versions in a fully automated way, benefitting from new features brought by the latest release.
+          </p>
         </li>
-        <li class="p-matrix__item">
-          <div class="p-matix__content">
-            <h5>Interoperability across various platforms</h5>
-            <p>
-              From hardware vendors to open source communities, Canonical
-              cooperates with various partners to ensure platform
-              interoperability and flexibility.
-            </p>
-          </div>
+        <li class="p-list__item">
+          <h3 class="p-heading--5">Interoperability across various platforms</h3>
+          <p>
+            From hardware vendors to open source communities, Canonical cooperates with various partners to ensure platform interoperability and flexibility.
+          </p>
         </li>
-        <li class="p-matrix__item">
-          <div class="p-matix__content">
-            <h5>Up to ten years of security updates</h5>
-            <p>
-              Canonical provides up to ten years of security updates under the
-              <a href="/security/esm">Expanded Security Maintenance (ESM)</a>
-              programme available for enterprise customers.
-            </p>
-          </div>
+        <li class="p-list__item">
+          <h3 class="p-heading--5">Up to ten years of security updates</h3>
+          <p>
+            Canonical provides up to ten years of security updates under the <a href="/security/esm">Expanded Security Maintenance (ESM)</a> programme available for enterprise customers.
+          </p>
         </li>
       </ul>
-      <p>
-        <a href="/openstack" class="p-button--positive">Continue reading</a>
-      </p>
+    </div>
+
+    <div class="grid-row">
+      <div class="grid-col-start-large-5 grid-col-4 grid-col-start-medium-3 grid-col-medium-2">
```

Comment:
This looks strange on med screens:

<img width="784" height="328" alt="image" src="https://github.com/user-attachments/assets/bd94e7c1-28b4-4cfd-acd7-9835c1c42a73" />

Maybe it should be grid-col-medium-4 with no offset for med?
```suggestion
      <div class="grid-col-start-large-5 grid-col-4 grid-col-medium-4">
```
<img width="755" height="276" alt="image" src="https://github.com/user-attachments/assets/5084bf2e-4516-46af-8c4a-8723106ffbf6" />